### PR TITLE
create header

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/App.module.scss [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+@import './styles/colors';
 
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   font-family: 'Montserrat';
   font-weight: 400;
-  color: $font-color-main-text;
+  color: $dark;
```
Comment (vell28) [inline src/App.module.scss]: dark what? blue? green? black?

File: src/App.module.scss [hunk 2]
```diff
@@ -23,16 +23,22 @@ ol, ul {
 }
 
 a {
+  transition: .3s all;
```
Comment (vell28) [inline src/App.module.scss]: move it to constant. The animation time have to the same in project.

File: src/components/Header/Menu/Menu.module.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        justify-content: space-between;
+        transform: translateY(50%);
+
+        .menu_item {
+            & a {
+                @include fontSize(400, 16px, 19.5px);
+                color: $black;
+            }
+
+            &:first-letter {
```
Comment (vell28) [inline src/components/Header/Menu/Menu.module.scss:15]: before use new css properties check documentation. Is it suport all browsers?

File: src/components/Header/Menu/Menu.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import { menuName } from 'constants/menuName'
```
Comment (vell28) [inline src/components/Header/Menu/Menu.tsx:4]: if the constant will only be used here, move it next to Component.

File: src/components/Header/MiddleHeader/MiddleHeader.module.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    .logo img {
+        height: 82px;
+    }
+
+    .search {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
```
Comment (vell28) [inline src/components/Header/MiddleHeader/MiddleHeader.module.scss]: move it to mixin, I guess it will be reused

File: src/constants/menuName.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+export const menuName = [
+    { id: 'livingRoom', name: 'living room' },
```
Comment (vell28) [inline src/constants/menuName.ts:2]: if name uniq we don't need ID. we can rewrite it to Array<string>

File: src/styles/_fontMixin.scss [hunk 7]
```diff
@@ -0,0 +1,5 @@
+@mixin fontSize($w, $s, $h) {
```
Comment (vell28) [inline src/styles/_fontMixin.scss]: as i see there is  not only `Size` rename plz to `font` or `fontStyle`

File: src/App.module.scss (hunk 1 above)
Comment (Ksenaaa) [reply src/App.module.scss]: fixed

File: src/App.module.scss (hunk 2 above)
Comment (Ksenaaa) [reply src/App.module.scss]: fixed

File: src/components/Header/Menu/Menu.module.scss (hunk 3 above)
Comment (Ksenaaa) [reply src/components/Header/Menu/Menu.module.scss:15]: Yes, it's support all browsers.

File: src/components/Header/Menu/Menu.tsx (hunk 4 above)
Comment (Ksenaaa) [reply src/components/Header/Menu/Menu.tsx:4]: No, it's will be used in another Components, e.g. Main, Bottom...

File: src/components/Header/MiddleHeader/MiddleHeader.module.scss (hunk 5 above)
Comment (Ksenaaa) [reply src/components/Header/MiddleHeader/MiddleHeader.module.scss]: fixed

File: src/constants/menuName.ts (hunk 6 above)
Comment (Ksenaaa) [reply src/constants/menuName.ts:2]: discussed

File: src/styles/_fontMixin.scss (hunk 7 above)
Comment (Ksenaaa) [reply src/styles/_fontMixin.scss]: fixed

File: src/components/Header/Menu/Menu.tsx [hunk 8]
```diff
@@ -1,12 +1,16 @@
...
 
-export const Menu = () => {
+type Props = {
+    isActiveMenu?: boolean;
+}
+
+export const Menu:FC<Props> = ({ isActiveMenu }) => {
     return (
-        <div className={styles.wrapper}>
+        <div className={!isActiveMenu ? styles.wrapper : styles.wrapper_active_menu}>
```
Comment (valentinPortside) [inline src/components/Header/Menu/Menu.tsx]: use `cn`!

File: src/components/Header/MiddleHeader/MiddleHeader.module.scss [hunk 9]
```diff
@@ -6,19 +9,114 @@
...
+    .search {
+        @include center-position;
     }
+}
+
+@media screen and (max-width: 975px) {
+    .wrapper {
+        padding: 0 15px;
+
+        .selectedUser {
```
Comment (valentinPortside) [inline src/components/Header/MiddleHeader/MiddleHeader.module.scss:74]: use `camalecase` for all style name

**Review by valentinPortside — APPROVED** [review]

File: src/components/Header/Menu/Menu.tsx (hunk 8 above)
Comment (Ksenaaa) [reply src/components/Header/Menu/Menu.tsx]: fixed

File: src/components/Header/MiddleHeader/MiddleHeader.module.scss (hunk 9 above)
Comment (Ksenaaa) [reply src/components/Header/MiddleHeader/MiddleHeader.module.scss:74]: fixed

File: src/components/Header/Burger/Burger.module.scss [hunk 10]
```diff
@@ -8,40 +8,40 @@
...
     }
 
-    &_first_line_cross {
+    &FirstLineCross {
         top: 10px;
         transform: rotate(45deg);
     }
     
-    &_second_line_cross {
+    &SecondLineCross {
```
Comment (vell28) [inline src/components/Header/Burger/Burger.module.scss]: &SecondLineCross,
&FirstLineCross {
   top: 10px;
}

combine the sane styles

File: src/components/Header/Menu/Menu.module.scss [hunk 11]
```diff
@@ -26,44 +19,55 @@
...
+    .menuList {
+        max-width: 919px;
+        flex-direction: row;
+        transform: translateY(50%);
 
-        .menu_item {
+        .menuItem {
             & a {
-                @include fontStyle(400, 24px, 40px);
+                @include fontStyle(400, 16px, 19.5px);
```
Comment (vell28) [inline src/components/Header/Menu/Menu.module.scss]: round to nearest whole number

File: src/components/Header/Burger/Burger.module.scss (hunk 10 above)
Comment (Ksenaaa) [reply src/components/Header/Burger/Burger.module.scss]: fixed

File: src/components/Header/Menu/Menu.module.scss (hunk 11 above)
Comment (Ksenaaa) [reply src/components/Header/Menu/Menu.module.scss]: fixed

**Review by vell28 — APPROVED** [review]

**Pull request merged**